### PR TITLE
Update Steps to Repro for GET based open redirects

### PR DESCRIPTION
**Pull request by RRudder**

### CODE REVIEW

File: submissions/description/unvalidated_redirects_and_forwards/open_redirect/get_based/template.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ GET-based open redirects can result in reputational damage for the business as c
 
 ## Steps to Reproduce
 
-1. Using a browser, navigate to: {{URL}}
-1. Copy and modify the URI so that the URL redirects to `url=<http://www.bugcrowd.com/`
-1. Submit this in a new browser window and you should be redirected to the Bugcrowd website.
+1. Using a browser, navigate to the open redirect on the endpoint: {{URL_with_no_redirect}}
```
Comment (EvermoreBugcrowd) [inline submissions/description/unvalidated_redirects_and_forwards/open_redirect/get_based/template.md:15]: Oh, perhaps better wording here is "navigate to the vulnerable endpoint:"

'the open redirect on the endpoint:' is strange wording here considering that it does not include the redirect code at this point.

**Review by EvermoreBugcrowd — CHANGES_REQUESTED** [review]
Step 1 wording still needs some work.

**Review by EvermoreBugcrowd — APPROVED** [review]
Awesome, looks good to me now.

**Review by emma-ferguson — APPROVED** [review]
LGTM

**Comment by RRudder** [comment]
@ticidesign See if you can approve this

**Review by ticidesign — APPROVED** [review]
LGTM

**Review by ticidesign — APPROVED** [review]
LGTM

**Review by ticidesign — APPROVED** [review]
LGTM

**Comment by RRudder** [comment]
@evildaemond could you please approve this PR? Ta!

**Review by norsec0de — APPROVED** [review]
Looks good to me

**Pull request merged**